pkg/sdk/api/v1beta1/groupversion_info.go

```diff
@@ -18,16 +18,18 @@
 package v1beta1
 
 import (
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
+	// "sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
```

Comment:
```suggestion
	"sigs.k8s.io/controller-runtime/pkg/scheme"
```
I think we should still use `controller-runtime`s scheme builder.